File: B面向对象/#复习最新课件/unit2 java Class Implementation(更新部分)/java-source/2.1.1/point/Point2D.java
package point;
public class Point2D {
	
	private float x;
	private float y;
	
	public Point2D(float initialX, float initialY) {
		
		x = initialX;
		y = initialY;
	}
	
	public float getX() {
		
		return x;
	}
	
	public float getY() {
		
		return y;
	}	
	
	public static void main(String[] args) {
		
		Point2D pointOne = new Point2D(100.0f,200.0f);
		
		System.out.println("x = "+pointOne.getX()
												
				+"  y = "+pointOne.getY());
												

		
												
	}
}

File: B面向对象/#复习最新课件/unit1 Object-oriented foundation and class diagram（更新）/source/1.1.2/Point2D.cpp
  #include <iostream>
  #include "point2D.h"
  using std::cout;
  using std::endl;

  float Point2D::getX(){
      return x;
  }
  float Point2D::getY(){
	  return y;
  }

  Point2D::Point2D(float initialX, float initialY){

	  x = initialX;
	  y = initialY;
  }

  Point2D::Point2D(const  Point2D& p) {
      x = p.x;
      y = p.y;
      cout<<"执行了复制构造函数"<<endl;

  }


